src/rust/bitbox02/src/safeinputstring.rs

```diff
@@ -15,19 +15,20 @@
 extern crate alloc;
 use alloc::boxed::Box;
 
+const INPUT_STRING_MAX_SIZE: usize = bitbox02_sys::INPUT_STRING_MAX_SIZE as _;
+
 /// C-style including null terminator, as it is used in C only so far.
```

Comment:
It is also used in many parts in rust these days, like unlock_keystore > password::enter these days, without seemingly switching to C. Of course it does underneath. So, I guess still good to mention it's a null-terminated string but C only - that's probably questionable.

Reply:
Right! Removed that part.